src/ChainTransfer/ChainTransferBase.cs

```diff
@@ -175,4 +188,14 @@ void ScaleChildren(Node3D nodesContainer)
 			node.Scale = new Vector3(1 / owner.Scale.X, 1, 1);
 		}
 	}
+
+	private void EnsureValidNodeReferences()
```

Comment:
Why not use https://docs.godotengine.org/en/stable/classes/class_node.html#class-node-method-is-node-ready?

Reply:
Because I still want the setter to have an immediate effect. The alternative would be calling the setter again in `_Ready` to ensure it did it's job.

For this part it wouldn't really matter because `Active` is usually false on init anyway, but for other parts like the ConveyorAssemblies, we've had to really lean-into embracing setters' early nature. It was one of the tricks I had to use to make `Duplicate` not print errors when we had all those editable children instances. `_Ready` was too late.